### DISCUSSION TITLE
Facing issue in running pika

### DISCUSSION THREAD

**Discussion by muhammad-abrar-hussain** [pika/pika#1501] [Q&A] [answered]
/usr/bin/python3 /Users/macos/abrar/python_env/rabbitMQ/receiver.py 
Traceback (most recent call last):
  File "/Users/macos/abrar/python_env/rabbitMQ/receiver.py", line 2, in <module>
    import pika, sys, os
  File "/Library/Python/3.9/site-packages/pika/__init__.py", line 20, in <module>
    from pika.adapters import BaseConnection
  File "/Library/Python/3.9/site-packages/pika/adapters/__init__.py", line 43, in <module>
    from pika.adapters.libev_connection import LibevConnection
  File "/Library/Python/3.9/site-packages/pika/adapters/libev_connection.py", line 106
    self.async = None
         ^
SyntaxError: invalid syntax

**Comment by lukebakken** [comment]
Hello, thanks for using Pika.

Please provide a project via GitHub, that I can clone, with instructions for reproducing this issue. I can't assist you otherwise.

More than likely, this issue is specific to your development environment, or how you have set up your project.

**Reply by muhammad-abrar-hussain** [reply]
Hi @lukebakken, 

Thank you for your assistance. 

I'm installing Pika in a virtual environment using Python version 3.9.x to set up a sender and receiver queue on a local RabbitMQ server. However, I'm encountering the error mentioned above. I'm also installing other packages, like FastAPI.

**Reply by lukebakken** [reply]
Please provide a project via GitHub, that I can clone, with instructions for reproducing this issue. I can't assist you otherwise.

**Reply by muhammad-abrar-hussain** [reply]
https://github.com/Muhammad-Abrar-Hussain/pika-RabbitMQ

@lukebakken, you can access the repository using the link above. Pika and Python 3.9 are installed in the virtual environment named RMQ_env. I am facing an error while running the file receiver.py, as mentioned in the first message.

**Comment by shinji-s** [comment]
It appears to me that the pika version being used is way too old. 'async' has become a keyword since python3.5. You are using virtual-env incorrectly and a lurking old version is being pulled, perhaps?

**Reply by muhammad-abrar-hussain** [reply, marked as answer]
> It appears to me that the pika version being used is way too old. 'async' has become a keyword since python3.5. You are using virtual-env incorrectly and a lurking old version is being pulled, perhaps?

 am currently using the latest version of Pika, which is 1.3.2. The issue arose due to the file path of my receiver.py. I have fixed it, and everything is working fine now.

**Reply by muhammad-abrar-hussain** [reply]
Thankyou @lukebakken  and @shinji-s  for your time :)

**Reply by lukebakken** [reply]
@Muhammad-Abrar-Hussain nice work figuring it out! Please feel free to use https://github.com/pika/pika/discussions if you have further questions.
